frontend/src/@types/bot.d.ts

```diff
@@ -82,6 +93,7 @@ export type UpdateBotResponse = {
   instruction: string;
   description: string;
   embeddingParams?: EmdeddingPrams;
```

Comment:
Oops, this should be not nullable. Could you make it non-nullable both embeddingParams and generationConfig?